src/dra/843B0.c

```diff
@@ -2303,7 +2303,225 @@ void EntityAguneaHitEnemy(Entity* self) {
     }
 }
 
-INCLUDE_ASM("dra/nonmatchings/843B0", func_80129864);
+void func_80129864(Entity* s2) {
```

Comment:
```suggestion
void func_80129864(Entity* self) {
```
let's normalize the variable names